src/blockchain-scanner.service.ts

```diff
@@ -1,9 +1,10 @@
+import '@frequency-chain/api-augment';
 import { Logger, OnApplicationBootstrap } from '@nestjs/common';
 import { BlockHash } from '@polkadot/types/interfaces';
 import { OnQueueEvent, QueueEventsHost, QueueEventsListener } from '@nestjs/bullmq';
 import { BlockchainService } from './blockchain/blockchain.service';
 import { ReconnectionCacheMgrService } from './cache/reconnection-cache-mgr.service';
-import { ReconnectionServiceConstants } from './constants';
+import * as ReconnectionServiceConstants from './constants';
```

Comment:
Namespaces are now considered bad form and to use this pattern instead.

Reply:
Yeah, I've noticed that recently. Not sure if I agree with the thinking there--I think the name of the namespace should be determined by the author, not the importer. But that's a philosophical debate...